rockcraft.yaml

```diff
@@ -1,12 +1,14 @@
 ---
 name: sdcore-upf-bess
-base: ubuntu@20.04
+base: ubuntu@22.04
```

Comment:
For curiosity, does the change work with 24.04-dev also?

Reply:
Not completely, I need to start from this branch again from this branch that is known to work. It will definitely require some changes in packages version, and some additional changes to compiler flags, but I had issues with the linker, and I am not sure if this is an issue with rockcraft at this point.